sdk/datafactory/Azure.ResourceManager.DataFactory/src/Azure.ResourceManager.DataFactory.csproj

```diff
@@ -5,4 +5,7 @@
     <Description>Azure Resource Manager client SDK for Azure resource provider DataFactory</Description>
     <PackageTags>azure;management;arm;resource manager;datafactory</PackageTags>
   </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core.Expressions.DataFactory\src\Azure.Core.Expressions.DataFactory.csproj" />
```

Comment:
should be able to make this a package ref, and probably need to add the version to packages.data.props

Reply:
Generally we use project refs rather than put alpha versions in the packages.data.props file.

Reply:
this won't get checked in until we at least beta the new library, so I always add it in the way it would be in the end just to validate all the structure / ci stuff works appropriately.

Reply:
> this won't get checked in until we at least beta the new library

Well that wouldn't be the case if we used a project reference 😄